File: Mobile/IT_SUPPORTER_App/app/src/main/java/com/odts/it_supporter_app/utils/ResponseObject.java
package com.odts.it_supporter_app.utils;

import com.google.gson.annotations.SerializedName;

public class ResponseObject<T> {
    @SerializedName("ObjReturn")
    private T ObjReturn;

    @SerializedName("IsError")
    private boolean isError;

    @SerializedName("ErrorMessage")
    private String errorMessage;

    @SerializedName("WarningMessage")
    private String warningMessage;

    @SerializedName("SuccessMessage")
    private String successMessage;

    public ResponseObject(T objReturn) {
        ObjReturn = objReturn;
    }

    public T getObjReturn() {
        return ObjReturn;
    }

    public void setObjReturn(T objReturn) {
        ObjReturn = objReturn;
    }

    public boolean isError() {
        return isError;
    }

    public void setError(boolean error) {
        isError = error;
    }

    public String getErrorMessage() {
        return errorMessage;
    }

    public void setErrorMessage(String errorMessage) {
        this.errorMessage = errorMessage;
    }

    public String getWarningMessage() {
        return warningMessage;
    }

    public void setWarningMessage(String warningMessage) {
        this.warningMessage = warningMessage;
    }

    public String getSuccessMessage() {
        return successMessage;
    }

    public void setSuccessMessage(String successMessage) {
        this.successMessage = successMessage;
    }
}

File: AutoTool/Automatic_updating_of_seniority/Automatic_updating_of_seniority/ResponseObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Automatic_updating_of_seniority
{
    public class ResponseObject<Object>
    {
        public bool IsError { get; set; }
        public String ErrorMessage { get; set; }
        public String WarningMessage { get; set; }
        public String SuccessMessage { get; set; }
        public Object ObjReturn { get; set; }
    }
}
